src/components/questDisplay.html

```diff
@@ -1,11 +1,11 @@
 <div class="page-item" style="background-color: white; min-height: 100px">
     <br>
     <button style="vertical-align: middle;" class='btn btn-primary'
-            data-bind='visible: !player.currentQuest(),
+            data-bind='visible: (player.tutorialProgress() == 6) && !player.currentQuest(),
```

Comment:
Is 6 the final tutorial quest? A constant is better then, or maybe a function that counts them